docker/owlbot/python/cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+    - name: 'gcr.io/cloud-builders/docker'
+      args: [ 'build',
+        '-t', 'gcr.io/repo-automation-bots/owlbot-nodejs:$SHORT_SHA',
+        '-t', 'gcr.io/repo-automation-bots/owlbot-nodejs:latest',
+        '-f', 'docker/owlbot/nodejs/Dockerfile', '.' ]
+    - name: 'gcr.io/cloud-builders/docker'
+      args: ['push', 'gcr.io/repo-automation-bots/owlbot-nodejs:$SHORT_SHA']
```

Comment:
I replaced `nodejs` with `python` in this file. I successfully built the image but I'm unable to push. I received this error: `Step #1: denied: Token exchange failed for project 'repo-automation-bots'. Caller does not have permission 'storage.buckets.get'. To configure permissions, follow instructions at: https://cloud.google.com/container-registry/docs/access-control`

Reply:
Here is my `docker/owlbot/python/cloudbuild.yaml`:
```
# Copyright 2021 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
steps:
    - name: 'gcr.io/cloud-builders/docker'
      args: [ 'build',
        '-t', 'gcr.io/repo-automation-bots/owlbot-python:$SHORT_SHA',
        '-t', 'gcr.io/repo-automation-bots/owlbot-python:latest',
        '-f', 'docker/owlbot/python/Dockerfile', '.' ]

    - name: 'gcr.io/cloud-builders/docker'
      args: ['push', 'gcr.io/repo-automation-bots/owlbot-python:$SHORT_SHA']

    - name: 'gcr.io/cloud-builders/docker'
      args: ['push', 'gcr.io/repo-automation-bots/owlbot-python:latest']

```

Reply:
Thanks, I updated the cloudbuild.yaml in the PR to match yours.

What command line are you running when you see the error?

Reply:
```
gcloud builds submit --config=docker/owlbot/python/cloudbuild.yaml --substitutions=SHORT_SHA=$(git rev-parse --short HEAD) 
```